raster/src/main/scala/geotrellis/raster/summary/MultibandTileSummaryMethods.scala

```diff
@@ -0,0 +1,73 @@
+package geotrellis.raster.summary
+
+import geotrellis.raster._
+import geotrellis.raster.histogram.{Histogram, StreamingHistogram}
+import geotrellis.util.MethodExtensions
+
+
+/**
+  * Trait containing [[MultibandTile]] extension methods for summaries.
+  */
+trait MultibandTileSummaryMethods extends MethodExtensions[MultibandTile] {
+
+  /**
+    * Contains several different operations for building a histograms
+    * of a raster.
+    *
+    * @note     Tiles with a double type (FloatConstantNoDataCellType, DoubleConstantNoDataCellType) will have their values
+    *           rounded to integers when making the Histogram.
+    */
+  def histogram: Array[Histogram[Int]] =
+    self.bands.map(_.histogram).toArray
+
+  /**
+    * Create a histogram from double values in a raster.
+    */
+  def histogramDouble(): Array[Histogram[Double]] =
+    self.bands.map(_.histogramDouble(StreamingHistogram.DEFAULT_NUM_BUCKETS)).toArray
```

Comment:
This should just pass to `histogramDouble(StreamingHistogram.DEFAULT_NUM_BUCKETS)` - it's a good practice to only let the last overload do any actual work, so that code changes will only have to ever happen there.